praw/models/reddit/comment.py

```diff
@@ -13,6 +9,10 @@
     UserContentMixin,
 )
 from .redditor import Redditor
+from ..comment_forest import CommentForest
```

Comment:
Why were these moved?

Reply:
I accidentally used autoformat with my IDE. I can undo these changes, but they really do not matter.